packages/astro/src/core/errors/errors-data.ts

```diff
@@ -18,18 +18,6 @@ export interface ErrorData {
  */
 // Astro Errors, most errors will go here!
 
-/**
- * @docs
```

Comment:
Just checking whether we are deleting, or deprecating error messages that only ever occurred for experimental features!

Reply:
We are deleting on purpose